packages/flame_spine/lib/src/skeleton_render.dart

```diff
@@ -0,0 +1,174 @@
+  }
+
+  bool get triangleRendering => _renderObject.triangleRendering;
+
+  set triangleRendering(bool value) {
+    _renderObject.triangleRendering = value;
+  }
+
+  /// Нow many percent increase the size of the animation
+  /// relative to the size of the first frame.
+  double get frameSizeMultiplier => _renderObject.frameSizeMultiplier;
+
+  set frameSizeMultiplier(double value) {
+    _renderObject.frameSizeMultiplier = value;
+  }
+
+  /// A start animation. We will use it for calculate bounds by frames.
+  String? get animation => _renderObject.animation;
+
+  set animation(String? value) {
+    _renderObject.animation = value;
+  }
+
+  void advance(double dt) {}
+
+  void render(Canvas canvas, Size size) {
+    final bound = _renderObject.bounds;
+
+    if (bound == null) {
+      throw 'SkeletonRender was rendered but Bounds are not present ';
+    }
+
+    _paint(canvas, bound, size);
+  }
+
+  void destroy() {
+    _renderObject.dispose();
+  }
+
+  void _paint(Canvas canvas, Bounds bounds, Size size) {
+    final contentHeight = bounds.size.y;
+    final contentWidth = bounds.size.x;
+    final x = -bounds.offset.x -
+        contentWidth / 2.0 -
+        (alignment.x * contentWidth / 2.0);
+    final y = -bounds.offset.y -
+        contentHeight / 2.0 +
+        (alignment.y * contentHeight / 2.0);
+
+    var scaleX = 1.0, scaleY = 1.0;
+
+    switch (boxFit) {
+      case BoxFit.fill:
+        scaleX = size.width / contentWidth;
+        scaleY = size.height / contentHeight;
+        break;
+      case BoxFit.contain:
+        final double minScale =
+            math.min(size.width / contentWidth, size.height / contentHeight);
+        scaleX = scaleY = minScale;
+        break;
+      case BoxFit.cover:
+        final double maxScale =
+            math.max(size.width / contentWidth, size.height / contentHeight);
+        scaleX = scaleY = maxScale;
+        break;
+      case BoxFit.fitHeight:
+        final minScale = size.height / contentHeight;
+        scaleX = scaleY = minScale;
+        break;
+      case BoxFit.fitWidth:
+        final minScale = size.width / contentWidth;
+        scaleX = scaleY = minScale;
+        break;
+      case BoxFit.none:
+
+      case BoxFit.scaleDown:
+        final double minScale =
+            math.min(size.width / contentWidth, size.height / contentHeight);
+        scaleX = scaleY = minScale < 1.0 ? minScale : 1.0;
+        break;
+
+      default:
+        scaleX = scaleY = 1.0;
+        break;
+    }
+
+    canvas
+      ..translate(
+        size.width / 2.0 + (alignment.x * size.width / 2.0),
+        size.height / 2.0 + (alignment.y * size.height / 2.0),
+      )
+      ..scale(scaleX, -scaleY)
+      ..translate(x, y);
+
+    _renderObject.draw(canvas);
+
+    canvas.restore();
+  }
+}
+
+Future<SkeletonAnimation> loadSkeleton(String name) async {
+  const pathPrefix = 'assets/';
+  return SkeletonAnimation.createWithFiles(name, pathBase: pathPrefix);
```

Comment:
this function loads file from flutter/assets which is underlying lib spine_flutter.
it not allowing to have tests inside test/assets folder.